app/controllers/companies_controller.rb

```diff
@@ -2,21 +2,14 @@ class CompaniesController < ApplicationController
 
   def new
     @company = Company.new
-    @admin = @company.admins.build
+    @admins = @company.admins.build
```

Comment:
building admin as i'm using nested attributes for in it and for obtaining admin's fields i'll have to build them before.